arnoldPlugins/gaffer.mtd

```diff
@@ -1046,6 +1075,22 @@
 
 	gaffer.nodeMenu.category STRING "Texture"
 
+	gaffer.graphEditorLayout.defaultVisibility BOOL false
+
+	gaffer.layout.activator.densityEnabled STRING "parameters['pattern'].getValue().startswith('cell')"
+	gaffer.layout.activator.coordSpaceIsPref STRING "parameters['coord_space'].getValue() == 'Pref'"
+
+	[attr P]
+		gaffer.graphEditorLayout.visible BOOL true
+	[attr offset]
+		gaffer.graphEditorLayout.visible BOOL true
+	[attr color]
+		gaffer.graphEditorLayout.visible BOOL true
+	[attr density]
+		gaffer.layout.activator STRING "densityEnabled"
```

Comment:
Just hanging this comment here for want of a better place. Looking through all the boolean parameters in Arnold, these ones seem like they might be good candidates to be activators :

```
light_decay.parameters.use_near_atten
light_decay.parameters.use_far_atten
*_light.parameters.cast_shadows
triplanar.cell
```

Reply:
Fixed in 6effe8fb9962734b64e353cd6e28c3edd27a75a9